docs/orchestrating-elastic-stack-applications/agent.asciidoc

```diff
@@ -0,0 +1,403 @@
+:page_id: elastic-agent
+:agent_recipes: https://raw.githubusercontent.com/elastic/cloud-on-k8s/{eck_release_branch}/config/recipes/elastic-agent
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Run Elastic Agent on ECK
+
+experimental[]
+
+This section describes how to configure and deploy Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] with ECK.
+
+NOTE: Please note that link:https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation.html[enrollment in Fleet] is currently not supported.
```

Comment:
Right now it sounds a bit like enrolment is not supported, but Fleet is, so maybe the below (or something similar).

```suggestion
NOTE: Please note that using link:https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation.html in [Fleet] to manage your Elastic Agents is currently not supported.
```